RealmModelGenerator/AttributesRelationshipsVC.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  AttributesRelationshipsViewController.swift
+//  RealmModelGenerator
+//
+//  Created by Zhaolong Zhong on 3/31/16.
+//  Copyright © 2016 QuarkWorks. All rights reserved.
+//
+
+import Cocoa
+
+protocol AttributesRelationshipsVCDelegate: class {
+    func attributesRelationshipsVC(attributesRelationshipsVC:AttributesRelationshipsVC, selectedAttributeDidChange attribute:Attribute?)
+    func attributesRelationshipsVC(attributesRelationshipsVC:AttributesRelationshipsVC, selectedRelationshipDidChange relationship:Relationship?)
+}
+
+class AttributesRelationshipsVC: NSViewController, AttributesVCDelegate, RelationshipsVCDelegate {
+    static let TAG = NSStringFromClass(AttributesRelationshipsVC)
+    
+    let attributesVC = NSStoryboard(name: "Main", bundle: nil).instantiateControllerWithIdentifier("AttributesViewController") as! AttributesViewController
+    let relationshipsVC = NSStoryboard(name: "Main", bundle: nil).instantiateControllerWithIdentifier("RelationshipsViewController") as! RelationshipsViewController
```

Comment:
there is a property on NSViewController that is called storyboard. you should use this instead of rebuilding the NSStoryboard file over again.